### ISSUE TITLE
search dialog title should be h1

### ISSUE THREAD

**Issue by ronnymikalsen** [NationalLibraryOfNorway/ngx-mime#372]
**Describe the bug**
Search dialog title should be h1 to meet material design and accessibility criterias

**To Reproduce**

1.  Start demo app
2.  Open Searchable manifest
3.  Open search dialog
4.  Inspect title and see error

**Expected behavior**
Search dialog title should be h1

**Screenshots**
![image](https://user-images.githubusercontent.com/10496433/144005758-ea5a71cc-bf53-4e9a-b972-4f5a1afa7af7.png)

**Which versions of NGX-Mime, Angular, Material, OS, TypeScript, browsers are affected?**
v12.4

**Additional context**
